projects/igniteui-angular/src/lib/banner/banner.component.ts

```diff
@@ -0,0 +1,165 @@
+import { Component, NgModule, EventEmitter, Output, Input, ViewChild, ElementRef, ContentChild } from '@angular/core';
+import { IgxExpansionPanelModule } from '../expansion-panel/expansion-panel.module';
+import { AnimationSettings } from '../expansion-panel/expansion-panel.component';
+import { BrowserModule } from '@angular/platform-browser';
```

Comment:
Don't import `BroweserModule` directly in our components. This is an application level import and will break the user's app when this module is lazy loaded for example.

Reply:
Removed in the latest commit